scripts/label_for_url.py

```diff
@@ -298,6 +298,26 @@ def determine_label(input_url: str) -> str:
         )
         page_id = f"/{m.group('pageId')}" if m.group("pageId") else ""
         return f"{space}{title}{anchor}" or f"{m.group('host')}{page_id}"
+    elif m := re.search(
+        r"^https://(?P<org>.*atlassian.*)"
```

Comment:
🐛 Incompatible with line 320 -- use the same value in both places

`host` makes more sense to me; we _could_ do `(?P<org>.*?)\.atlassian\.(?:net|com)`, I guess, if we wanted to use only the subdomain in the fallback case?